src/lib/accessibility/Behaviors/Chat/ChatBehavior.ts

```diff
@@ -0,0 +1,31 @@
+import { Accessibility, FocusZoneMode } from '../../interfaces'
+import * as keyboardKey from 'keyboard-key'
+import { FocusZoneDirection } from '../../FocusZone'
+
+const CHAT_FOCUSZONE_ATTRIBUTE = 'chat-focuszone'
+
+const ChatBehavior: Accessibility = {
+  attributes: {
+    root: {
+      role: 'presentation',
+    },
+  },
+  focusZone: {
+    mode: FocusZoneMode.Wrap,
+    props: {
+      shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.ArrowRight,
+      direction: FocusZoneDirection.vertical,
+      defaultActiveElement: `[${CHAT_FOCUSZONE_ATTRIBUTE}] > * > *:last-child`, // select last chat message by default
```

Comment:
Whenever a chat is rendered, focus is set to it's last element.
That means that playing with chat in docsite is hardly usable.
Plus is this the behavior we want in the real app?